extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -133,26 +133,44 @@
 .i-amphtml-lbg-desc-box.standard {
   background: linear-gradient(rgba(0,0,0,0), rgba(0,0,0,0.5));
   max-height: 6rem !important;
+  transition: max-height ease-out 0.3s !important;
 }
 
 .i-amphtml-lbg-desc-box.overflow {
-  background-color: rgba(0,0,0,0.5);
   overflow-y: auto !important;
   -webkit-overflow-scrolling: touch !important;
-  top: 0px !important;
-  padding-top: 50px !important;
+  max-height: 100% !important;
+  transition: max-height ease-in 0.3s !important;
+}
+
+.i-amphtml-lbg-desc-mask {
+  width: 100% !important;
+  position: fixed !important;
+  bottom: 0 !important;
 }
 
 .i-amphtml-lbg-desc-text {
   padding: 20px !important;
   position: relative !important;
 }
 
-.i-amphtml-lbg-desc-text.non-expanded {
-  overflow-y: hidden !important;
-  overflow-x: hidden !important;
-  text-overflow: ellipsis !important;
-  white-space: nowrap !important;
+.i-amphtml-lbg-desc-box.overflow .i-amphtml-lbg-desc-text {
+  padding-top: 40px !important;
+}
+
+.i-amphtml-lbg-desc-box.standard .i-amphtml-lbg-desc-mask {
```

Comment:
Added a mask component. The purpose of this component is to shim / cover the entire lightbox. If the UX decision is only to shim to the height of the description text, we may not need this at all.